plugins/importexport/native/filter/SubmissionNativeXmlFilter.inc.php

```diff
@@ -98,8 +98,6 @@ function createSubmissionNode($doc, $submission) {
 		$workflowStageDao = DAORegistry::getDAO('WorkflowStageDAO'); /** @var $workflowStageDao WorkflowStageDAO */
 		$submissionNode->setAttribute('stage', WorkflowStageDAO::getPathFromId($submission->getData('stageId')));
 
-		// FIXME: language attribute (from old DTD). Necessary? Data migration needed?
```

Comment:
Should it be removed from https://github.com/pkp/pkp-lib/blob/stable-3_3_0/plugins/importexport/native/pkp-native.xsd#L268 also?

Reply:
For the stable branch I decided to leave this attribute forgotten, just to avoid creating a breaking change on the validation of old exports.
FYI I've already removed this at the `main` branch previously.

Reply:
I remember about the main branch.  

Maybe you are right. No harm in leaving it there (the code does not deal with that anyway)